ipfs-cluster-ctl/main.go

```diff
@@ -105,11 +107,30 @@ func main() {
 			Name:  "debug, d",
 			Usage: "set debug log level",
 		},
+		cli.StringFlag{
+			Name: "credentials, c",
```

Comment:
perhaps rename to --basic-auth (or even `-u, --user USER[:PASSWORD]  Server user and password` like `curl` does) . Not sure if it deserves a short-hand alias. I'd like to keep -c for config (in line with server).

Reply:
Hm, so right now (I think) an empty password would work by passing `--basic-auth <username>:` (note the `:` at the end) but you'd get an error if you just passed `--basic-auth <username>` (no `:`). Might be nice to support the latter instead, I'll make that change.

Reply:
sounds good